cypress/integration/pages/topicPage/tests.js

```diff
@@ -60,7 +51,6 @@ export default ({ service, pageType, variant }) => {
         });
         cy.log(`topic id ${topicId}`);
       }
-      cy.clearLocalStorage();
```

Comment:
Oh, do we not need this anymore?

Reply:
This blog post: https://www.cypress.io/blog/2023/03/08/test-isolation-as-a-best-practice/ under the `Disabling Test Isolation ≠ Pre-v12` heading seems to indicate that pre v12 Cypress (which we use) already automatically clear local storage. So the above is not needed unless used in a specific testcase.

Reply:
Sweet! Thanks for explaining